racun-service/src/main/java/racun/repository/RacunRepository.java

```diff
@@ -0,0 +1,15 @@
+package racun.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import racun.model.Racun;
+
+
+public interface RacunRepository extends JpaRepository<Racun, Long>{
+
+    @Query("SELECT R FROM Racun R WHERE R.brojRacuna=:broj")
```

Comment:
Ove Query anotacije ti generalno ne trebaju pošto Hibernate može sam da generiše query po imenu funkcije.